### DISCUSSION TITLE
혹시 CSS 라이브러리를 emotion으로 바꾸신 이유가 있을까요?

### DISCUSSION THREAD

**Discussion by khakhiD** [morethanmin/morethan-log#363] [Q&A]
안녕하세요. @morethanmin 님.
작년에 포크해서 이것 저것 수정하고 뚝딱여서 블로그로 잘 사용하고 있습니다.
tailwind CSS를 제 입맛에 맞게 몇 가지 수정하다 원본 코드가 어떻게 되어 있었는지 궁금해서 들어왔습니다.
기존에는 `tailwind`를 사용하셨던 걸로 기억하는데, CSS-in-JS `emotion`으로 라이브러리를 바꾸신 것 같더라구요.
혹시 변경하시게 된 이유가 있을까요?

> p.s. 항상 감사히 잘 사용하고 있습니다.

**Comment by morethanmin** [comment]
@khakhiD 

특별한 기술적 이유가 있어서는 아니고, 과거 개인적으로 사용하던 시절에 이것저것 적용해보면서 바꾸게 되었습니다. (지금은 바꾸면 안될 것 같네요 😂 )

+ 전환하면서 스타일링 관련해서 미흡한 부분들이 존재하는데, 조만간 작업해두겠습니다~!
